Gordon360/Extensions/DateTimeExtensions.cs

```diff
@@ -3,6 +3,32 @@ namespace Gordon360.Extensions.System;
 
 static public class DateTimeExtensions
 {
+    /// <summary>
+    /// Casts a TimeOnly to a DateTime with a given start date at 1/1/2000
+    ///  2000 is a clean turn of the millennium:
+    ///  this datetime falls under:
+    ///      min javascript datetime (1/1/1970)
+    ///      min sql datetime (1/1/1753)
+    ///      min c# datetime (1/1/0001)
```

Comment:
Why add another date to the existing confusion that you reference? I would choose one of the existing ones and stick with that so that you don't make the problem worse :)